config/prometheus/role.yaml

```diff
@@ -0,0 +1,46 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: prometheus-k8s
+  namespace: system
```

Comment:
Why do we create `Role` in `system` namespace? What is `system` namespace for?

Reply:
well, this is where kustomize works, refer to https://github.com/kubernetes-sigs/kueue/blob/a4703d33fe40045629340b8cf5de4bbcf1d17340/config/default/kustomization.yaml#L4-L9

Reply:
Got it. Thanks